html/elements/optgroup.json

```diff
@@ -114,13 +114,13 @@
                 "version_added": true
               },
               "firefox": {
-                "version_added": true
+                "version_added": "2"
               },
               "firefox_android": {
                 "version_added": true
               },
               "ie": {
-                "version_added": false
+                "version_added": "6"
```

Comment:
What makes you say it is supported since IE 6 and not earlier? "6" and  not true here?

Reply:
In all cases, these numbers come from caniuse. If you'd prefer I just leave things with true/false I can do so; I don't know if we consider that a canonical source of this information, now that I think of it.

Reply:
I dug a bit: IE5 for Mac was supporting it, but it is a different rendering engine, long dead. So I'm not sure about "6" being the first one. Please put true there.